test/test_hpyerr.py

```diff
@@ -33,8 +36,15 @@ def test_FatalError(self):
             @EXPORT(f)
             @INIT
         """)
-        # Calling mod.f() gives a fatal error, ending in abort().
-        # How to check that?  For now we just check that the above compiles
+        result = subprocess.run([
```

Comment:
Inside app-level tests the `sys` module has no `executable` attribute, so that provides a convenient way to skip the remainder of the test.

I don't understand app-level tests well enough to know if a better plan can be made -- a separate process probably needs to be involved somehow since although `SIGABRT` can be "caught", restoring the process back to a sane place seems likely to require crazy crazy hacks.